### PR TITLE
Code samples for validatepassword API

### PR DESCRIPTION
**Pull request by evigian**

### CODE REVIEW

File: python/validatepassword.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+PARAMS = {
+    "action": "validatepassword",
+    "format": "json",
+    "password": "your_password",
+}
+
+R = S.post(url, data=PARAMS)
+DATA = R.json()
+
+print(DATA)
```
Comment (brendajerop) [inline python/validatepassword.py]: The python code seems to be written twice in this file

Comment (evigian) [reply python/validatepassword.py]: The second example contains an additional parameter for creating a new user. Should I remove it from Python and from the other files as well?

**Comment by evigian** [comment]
Here's the link to my Sandbox page:
https://www.mediawiki.org/wiki/User:Evigian/Sandbox/API:Validatepassword

File: python/validatepassword.py (hunk 1 above)
Comment (brendajerop) [reply python/validatepassword.py]: @evigian Since the difference is in only one parameter, you may remove it :)

Comment (evigian) [reply python/validatepassword.py]: Ok, I removed all of them. Thank you for reviewing it @brendajerop :)

File: javascript/validatepassword.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+    validatepassword.js
+
+    MediaWiki Action API Code Samples
+    Demo of `validate password` module: Validate a password against the wiki's password policies.
```
Comment (brendajerop) [inline javascript/validatepassword.js]: Let's make this " Demo of `Validatepassword` module: Validate a password against the wiki's password policies." Make this change if the other files as well

File: mediawikijs/validatepassword.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/*
+    validatepassword.js
+
+    MediaWiki Action API Code Samples
+    Demo of `validate password` module: Validate a password against the wiki's password policies.
+    MIT license
+*/
+
+# Validate a password for a current user.
```
Comment (brendajerop) [inline mediawikijs/validatepassword.js]: Remove this comment as it's confusing

File: python/validatepassword.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    validatepassword.py
+
+    MediaWiki Action API Code Samples
+    Demo of `validate password` module: Validate a password against the wiki's password policies.
+    MIT license
+"""
+
+import requests
+
+url = "https://en.wikipedia.org/w/api.php"
```
Comment (brendajerop) [inline python/validatepassword.py]: This variable name should be capitalized since it's a constant.

**Comment by brendajerop** [comment]
@evigian Thank you for making the changes! I'm still getting some lint errors with php. Please resolve the merge conflicts as well :)

**Comment by evigian** [comment]
Ok @brendajerop I resolved the conflict and added 2 semicolons that were missing from php file. Please let me know if there's any other issue :)

**Pull request merged**